ckan/controllers/user.py

```diff
@@ -371,6 +390,27 @@ def _save_new(self, context):
             context['message'] = data_dict.get('log_message', '')
             captcha.check_recaptcha(request)
             user = get_action('user_create')(context, data_dict)
+
+            if request.params['project_of_interest'] == 'Australian Microbiome':
+                username = user['name']
+
+                base = os.environ.get('LOCAL_CKAN_API_URL')
+                ckan_api_key = os.environ.get('CKAN_API_KEY')
+                remote = ckanapi.RemoteCKAN(base)
+
+                data = {
+                    'id': 'australian-microbiome',
+                    'username': username,
+                    'role': 'member'
+                }
+
+                remote.call_action(
+                    'organization_member_create',
+                    data_dict=data,
+                    apikey=ckan_api_key,
+                    requests_kwargs={'verify': False}
+                )
+
```

Comment:
no need to use the CKAN API, we're running within CKAN - just make a direct call to the relevant CKAN code :)

Reply:
Ah, okay - that makes sense, thanks Sam!